src/identifiers.md

```diff
@@ -60,4 +63,34 @@ See the [HIR chapter][hir-map] for more detailed information.
 
 ## In the MIR
 
-**TODO**
+- [`BasicBlock`] identifies a *basic block*. It points to an instance of
+  [`BasicBlockData`].
+
+- [`Local`] identifies a local variable in a function. Its associated data is in
+  [`LocalDecl`].
+
+- [`Field`] identifies a struct's, union's, or enum variant's field. It is used
+  as a "projection" in [`Place`].
+
+- [`SourceScope`] identifies a name scope in the original source code.
+
+- [`Promoted`] identifies a promoted constant (related to const evaluation).
```

Comment:
It identifies a promoted *within* another item. So you always need a `GlobalId` to be able to identify a promoted across MIR bodies